spec/classes/init_spec.rb

```diff
@@ -2045,6 +2047,25 @@
     end
   end
 
+  context 'with sshd_config_maxauthtries not being a valid number' do
```

Comment:
To get this to work, we need to validate the parameter in the ssh class. Something like

`validate_integer()`